frontend/src/views/web/challenge/edit-challenge/edit-challenge-date.html

```diff
@@ -0,0 +1,46 @@
+<section class="ev-md-container text-center rm-overflow-y">
```

Comment:
Can we please decrease the width of modal?

Reply:
This might be a silly question, but how can I decrease the width? I am completely new to CSS...

Reply:
You can add a width css in `challenge.scss` by adding some class name to the section tag.